boards/arm/s32k1xx/s32k148evb/src/s32k1xx_bringup.c

```diff
@@ -113,6 +117,26 @@ int s32k1xx_bringup(void)
       /* Register EEEPROM block device */
 
       s32k1xx_eeeprom_register(0, 4096);
+#endif
```

Comment:
The indentation hear and below do not match.  Upon viewing the file, I can see that the indentation at lines 117-119 are wrong, not the new code.

Reply:
Thanks for pointing out, I realized it's also wrong in the other board files, unfortunely nxstyle didn't pick it up. I've fixed it now.